### PR TITLE
REF: IndexOpsMixin wrapping

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -4775,7 +4778,7 @@ def _convert_dtypes(
 
     @doc(NDFrame.isna, klass=_shared_doc_kwargs["klass"])
     def isna(self) -> "Series":
-        return super().isna()
+        return generic.NDFrame.isna(self)
```
Comment (jreback) [inline pandas/core/series.py:4781]: wouldn't you directly cal the one in missing itself?

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/core/series.py (hunk 1 above)
Comment (jreback) [reply pandas/core/series.py:4781]: actually doing the opposite for duplicated, so i think ok to in-line it here for consistency

Comment (jbrockmendel) [reply pandas/core/series.py:4781]: we're specifically inheriting the doc above

**Comment by jbrockmendel** [comment]
gentle ping; this has become a blocker for mypy fixups

**Review by jreback — APPROVED** [review]

File: pandas/core/series.py (hunk 1 above)
Comment (jreback) [reply pandas/core/series.py:4781]: ok pls consider to unfiy the calling covention with other methods , e.g. we either in-line things *here* or in a mixin or whatever to be consistent.

**Pull request merged**